### ISSUE TITLE
Fasterq-dump pathing issues

### ISSUE THREAD

**Issue by merrick893** [ncbi/sra-tools#810]
I have been working through using the SRA toolkit the past couple days. Me and my lab are hoping on downloading some 10x genomics single cell RNA-seq data to analyze but ran into issues downloading the fastq files directly by using the SRR accession number. Instead we were able to download the SRA file and are now trying to use the fasterq-dump command directly on the local SRA file. Every time I run the following code I run into the following error: 

![image](https://user-images.githubusercontent.com/131307592/233200059-cae451c1-8549-4850-a4b1-209ab56e14c8.png)

In this image I am in the bin directory of the SRA toolkit download and am trying to run the fasterq-dump command on a file in a different directory. I verified this is the correct path to the SRA file. The same error occurs even if I use the explicit SRA file path, changing the current directory, or using fastq-dump instead with the proper modifiers. For this example I turned off remote access to ensure the program used the local file. I am working on a MS Windows OS.

**Comment by merrick893** [comment]
Thank you for this insight! I will try prefetch on the accession and then fasterq-dump. If that doesn't work I'll try the linux VM. I'll be back with results.

**Comment by wraetz** [comment]
Another problem here:
the object you try to convert to FASTQ is in ..\..\NCBI_1\sra
the tool tries to 'cd' into this directory - and fails - it does not exist.
Are you sure that the object is in C:\Users\schol\Documents\BrownLab\SRAToolkit\NCBI_1\sra' ?
You are going 2 directories back in the path. Maybe you need more '..' in your relative path.

**Comment by merrick893** [comment]
Thanks for the response; however I am certain this is the correct directory. I couldn't find a convenient way to take a screenshot of this but this is the best I could do:

![image](https://user-images.githubusercontent.com/131307592/233431078-babb8f9a-193e-4b62-aec5-fb30ab66f007.png)


However, if as you said before this function is supposed to work on a directory of files instead of just a single file then it may be trying to cd into the SRR12024267.sra file which is not a directory.

**Comment by wraetz** [comment]
No the error you are seeing comes from the path you are giving being not valid. Try to give an absolute path instead of the relative one. Before doing this try it with "dir C:\the\path\to\the\file" in CMD.exe - Check if dir can resolve it.

**Comment by merrick893** [comment]
Here is the result of the dir resolution and followup fasterq-dump:

![image](https://user-images.githubusercontent.com/131307592/233452008-41781c36-18c1-4bc2-83ef-a069653e8da9.png)

It seems that the dir was able to find the directory but fasterq-dump was having issues

**Comment by wraetz** [comment]
Yes I see that there is a problem. We have to investigate that. Thank you for helping with the investigation.
Here is something you can try before going the WSL-Linux route:
In the bin-directory there is a file called fasterq-dump-orig.exe. Use that one instead of fasterq-dump.exe

**Comment by merrick893** [comment]
This solution looks like it is working!

The command still resulted in an error but by looking at the description it looks like it doesn't have required files in the same directory like you mentioned before. I will prefetch and ensure the correct dependencies are in the same directory as the sra file.

Thank you for your help in solving this issue!
